main.go

```diff
@@ -95,6 +95,8 @@ func main() {
 	}()
 
 	// Routing handler
+	http.Handle("/", http.FileServer(http.Dir("static_files")))
```

Comment:
I actually have done something a bit different in my branch to ensure other endpoints other than `/events/` and `/rooms/` do not work - or rather return 404. Why do we need this `http.FileServer(http.Dir("static_files"))` ?

Reply:
It will serve directly the files on path './static_files'
It's just easier to test (and provide other imgs, css, etc...)

Thus, http://localhost:8080/ will serve  'static_files/index.html'

Reply:
Hum, ok. Not sure if we need this now though, but sure. We must not forget to remove this in an upcoming version.